matching-funds/README.md

```diff
@@ -0,0 +1,24 @@
+# Matching Funds
+
+This application let uses create a matching fund campaing for their favorite charity.
```

Comment:
```suggestion
This application lets users create a matching fund campaign for their favorite charity.
```

Reply:
You have no idea how many times I misspelled `campaign` as `campaing` while developing this